Directory.Packages.props

```diff
@@ -43,7 +43,7 @@
       See https://github.com/open-telemetry/opentelemetry-dotnet/issues/5973 for more details.
   -->
   <ItemGroup>
-    <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="[10.0.0-rc.2.25502.107,)" />
+    <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="[10.0.0,)" />
```

Comment:
Consider Changelog entry. I am not sure if you discussed it on SIG.

Reply:
I recommend reviewing the previous CHANGELOG.md and RELEASENOTES.md files from the last .NET major release.

Reply:
Not sure what you mean - I've already pushed a commit to add an entry to all the CHANGELOG files for this PR.

Reply:
Done.